client/signup/steps/dss/index.jsx

```diff
@@ -22,18 +21,20 @@ const debug = debugFactory( 'calypso:dss' );
 export default React.createClass( {
 	displayName: 'DssThemeSelection',
 
+	propTypes: {
+		themes: React.PropTypes.array,
+		useHeadstart: React.PropTypes.bool,
+	},
+
 	getDefaultProps() {
 		return {
 			themes: [
-				'Boardwalk',
-				'Cubic',
-				'Edin',
-				'Cols',
-				'Minnow',
-				'Sequential',
-				'Penscratch',
-				'Intergalactic',
-				'Eighties'
+				{ name: 'Sela', slug: 'sela' },
+				{ name: 'Goran', slug: 'goran' },
+				{ name: 'Twenty Fifteen', slug: 'twentyfifteen' },
+				{ name: 'Sequential', slug: 'sequential' },
+				{ name: 'Colinear', slug: 'colinear' },
+				{ name: 'Edin', slug: 'edin' },
```

Comment:
The ones we were using were actually being passed in as props, but they were static, so I see no reason not to just put them here instead of in `steps.js`